### PR TITLE
3 - naked Yote 

### PR DESCRIPTION
**Pull request by ffugly**
Remove _all_ styles by default. The idea is to eliminate any implicit bias on the design side from app to app.  

With previous versions of Yote, we would run smack into the following issues a lot:
1. deeply nested SASS styles that were all over the place and impossible to find. which led to 
2. a built up library of base components, layouts, and styles that were esoteric and difficult to work on. which made it 
2. difficult to change any root components once implemented. which led us to avoid changing it. which led to 
2. the Bootstrap problem of all apps looking the same. 
 
We talked for years about different ways to solve these problems, none of which was perfect. These ranged from:
- picking a well supported library like Bootstrap, Material, or Semantic and just leaning into it. _Yet all apps would look the same_
- letting the developer use whatever CSS library they want. _Would be less usable by Fugitive, which would mean Fugitive would probably fall back into the above traps _
- building a YoteCSS utility library. _Very difficult work that would likely be esoteric and slow to evolve_

By integrating TailwindCSS we have a well documented utility CSS library with a huge following (which means it should remain well supported for a long time). However, what I'd like to avoid is the trap of building up another library of predefined layouts & styles that make it difficult to make fresh decisions from app to app. In order to do that, default, init-ed Yote should be a blank canvas with zero styling opinions. 

Starting bare bones is easier than trying to guess at the perfect version of some component and then inevitably have to go back and figure out why something is the way it is, and make a decision about how to change it -- especially for repeatable things like a form or a navigation element. With the blank canvas all of this is up to the designer & developer _for that project_. Use raw tailwind utility, buy and use tailwind ui, write your own scss. Doesn’t matter. 

Note: in order to keep things usable out of the box, I disabled Tailwind's normalizer so we can just let the browser render basic elements natively -- `<a>` tags, `<input>`, etc.  When ready to apply Tailwind styles, just have to flip a boolean in `tailwind.config` to get all that cross-browser goodness

### CODE REVIEW

File: web/src/global/components/navigation/DefaultNav.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+import { useHistory, useLocation } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+
+import { useGetLoggedInUser } from '../../../resources/user/authService';
+
+import { sendLogout } from '../../../resources/user/authStore';
+
+// import UI components
+import { Disclosure, Menu, Transition } from '@headlessui/react'
```
Comment (grantfowler) [inline web/src/global/components/navigation/DefaultNav.jsx]: remove these

**Review by grantfowler — CHANGES_REQUESTED** [review]
also remove UserProfile.jsx since its empty and unused

**Review by grantfowler — APPROVED** [review]

File: web/src/global/components/helpers/PageTabber.jsx [hunk 2]
```diff
@@ -52,91 +52,69 @@ const PageTabber = ({
...
               :
               null
             }
             {before.map((page, i) => (
-              <button
+              <a
                 key={`page-before-${i}`}
                 onClick={() => setPage(page)}
-                className={setPageBtnCommonClasses}
+                // className={setPageBtnCommonClasses}
```
Comment (WesJourdan) [inline web/src/global/components/helpers/PageTabber.jsx:72]: Probably want to remove this className and also remove all of the styling related stuff at the top of the file and the hero icon imports that are no longer being used.

File: web/src/resources/product/components/ProductListItem.jsx [hunk 3]
```diff
@@ -17,25 +16,23 @@ const ProductListItem = ({ id }) => {
...
 const Skeleton = () => {
   return (
-    <ListItem className="animate-pulse">
-      <p className="w-6/12 h-4 bg-gray-500"></p>
-      <p className="h-4"></p>
-      <p className="w-8/12 h-4 bg-gray-500"></p>
-    </ListItem>
+    <li className="animate-pulse">
+      <p>...</p>
+    </li>
```
Comment (WesJourdan) [inline web/src/resources/product/components/ProductListItem.jsx:35]: I think if we're going to keep ListItem.jsx then we should be using it here.

File: web/src/resources/user/components/UserLayout.jsx [hunk 4]
```diff
@@ -4,10 +4,19 @@
...
+    , title
+  } = props;
+  
+  // this can replace react-helmet if all we need to do is set the page title.
+  useEffect(() => {
+    document.title = title || "Yote App | User";
+  }, [title])
   return (
     <div className="container">
       <div className="container mx-auto">
```
Comment (WesJourdan) [inline web/src/resources/user/components/UserLayout.jsx]: Were these classNames left here on purpose?

**Review by WesJourdan — APPROVED** [review]

**Pull request merged**